mobile/src/calendar/calendar-screen.tsx

```diff
@@ -1,11 +1,12 @@
 import { createStackNavigator } from 'react-navigation';
 import React from 'react';
 import { CalendarScreenComponent } from './home-calendar-screen';
+import { transparentHeaderStackNavigatorConfig } from '../override/stack-navigator-config';
 
 //----------------------------------------------------------------------------
 export const CalendarScreen = createStackNavigator({
     Home: {
         screen: CalendarScreenComponent,
         path: '/',
     },
-});
+},                                                 transparentHeaderStackNavigatorConfig);
```

Comment:
What's up with formatting there?

Reply:
tslint ) Anyway, this changes are outdated.